lnbits/extensions/copilot/crud.py

```diff
@@ -71,15 +71,16 @@ async def update_copilot(
 ) -> Optional[Copilots]:
     q = ", ".join([f"{field[0]} = ?" for field in data])
     items = [f"{field[1]}" for field in data]
-    items.append(copilot_id)
-    await db.execute(f"UPDATE copilot.newer_copilots SET {q} WHERE id = ?", (items))
+    if copilot_id:
+        items.append(copilot_id)
+    await db.execute(f"UPDATE copilot.newer_copilots SET {q} WHERE id = ?", (items,))
     row = await db.fetchone(
         "SELECT * FROM copilot.newer_copilots WHERE id = ?", (copilot_id,)
     )
     return Copilots(**row) if row else None
```

Comment:
This function doesn't look like the `copilot_id` is actually optional. Won't this just crash if you don't pass it?

Reply:
changed it, ```copillot_id``` is now not ```Optional[str]```